atlascope/core/rest/endpoints/investigation_endpoints.py

```diff
@@ -89,3 +90,9 @@ def permissions(self, request, pk=None):
         payload = InvestigationDetailSerializer(investigation).data
         payload = {k: v for k, v in payload.items() if k in ['owner', 'investigators', 'observers']}
         return Response(payload, status=status.HTTP_200_OK)
+
+    @object_permission_required(edit_access=True)
+    @action(detail=True, methods=['GET'])
+    def pins(self, request, pk=None):
+        payload = {str(pin.id): PinSerializer(pin).data for pin in self.get_object().pins.all()}
```

Comment:
I think a list is preferable to a dictionary here. Maybe something like
`payload = PinSerializer(self.get_object().pins.all(), many=True).data`?

Reply:
I like this a lot better, too. Changed in 722402a